tls/s2n_resume.c

```diff
@@ -886,6 +884,8 @@ S2N_RESULT s2n_resume_encrypt_session_ticket(struct s2n_connection *conn, struct
 
     RESULT_GUARD_POSIX(s2n_aes256_gcm.io.aead.encrypt(&aes_ticket_key, &iv, &aad_blob, &state_blob, &state_blob));
 
+    conn->ticket_fields.key_intro_time = key->intro_timestamp;
+
```

Comment:
This seems like a very roundabout way to get access to the intro timestamp. Why can't we just get it from the key directly?